src/docs/arch-session.dox

```diff
@@ -0,0 +1,51 @@
+/*! @arch_page arch-session Session
+
+@section arch_session_def Definition
+After a @ref arch-connection has been established between the application and WiredTiger, the
+application can start sending requests to WiredTiger using a session. A session is internally
+represented by WT_SESSION and plays an important role since almost all operations require one.
```

Comment:
almost all operations are performed under the context of a session?

Reply:
See b9544afeedb411abcfede0d56b7eea041b152ec7